src/test/java/cloud/eppo/helpers/TestCaseValue.java

```diff
@@ -0,0 +1,63 @@
+package cloud.eppo.helpers;
+
+import cloud.eppo.ufc.dto.EppoValue;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class TestCaseValue extends EppoValue {
```

Comment:
Mostly not new code; Largely ported over from Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/androidTest/java/cloud/eppo/android/helpers/TestCaseValue.java))